content/W2015/W15-event-destress.md

```diff
@@ -0,0 +1,20 @@
+Title: Big CSters Discussion Circles: De-Stressing
+Date: 2015-03-05 18:00
+Category: Events
+Tags: social
+Slug: de-stress
+Author: Anna Lorimer
+Summary: School is stressful! Come find out what strategies upper&ndash;year students use to handle their workload.
```

Comment:
oh lol, this one should be a hyphen! Thanks for being so pro-active :D

For background on this dash snobbery of mine, see [this wikipedia article](http://en.wikipedia.org/wiki/Dash). Relevant excerpt is 

> The preference for an en dash instead of a hyphen in these coordinate/relationship/connection types of terms is a matter of style preference, not inherent orthographic "correctness"; both are equally "correct", and each is the preferred style in some style guides.

I prefer hyphens for these words over the n-dash. n-dashes are "correct" for ranges, however. And then m-dashes are the parenthetical ones.